docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.4"
+
+services:
+  authn_service:
+    image: cybercucumber/authn_service:latest
+
+  nginx:
+    container_name: nginx
```

Comment:
We need a way to access logs without connecting to nginx container. So probably we should use volumes here to mount host filesystem to nginx

Reply:
> We need a way to access logs without connecting to nginx container. So probably we should use volumes here to mount host filesystem to nginx

I have added volume to nginx container. Logs locate /home/authn/repo/nginx-logs. File authn.json.log